### PR TITLE
wiki - editor - add buttons 'inline code', 'empty checkbox', 'checked checkbox'

### PR DESCRIPTION
**Pull request by Cherrg**
affects #5436

## Images

grafik

Signed-off-by: Michael Gnehr <michael@gnehr.de>

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
Maintainer note: This has 2 LGTM, but as we are in freeze it'll have to 1.10.0-dev to merge

**Pull request merged**
